diffkemp/cli.py

```diff
@@ -12,6 +12,38 @@ def make_argument_parser():
     sub_ap = ap.add_subparsers(dest="command", metavar="command")
     sub_ap.required = True
 
+    # "build" sub-command
+    build_ap = sub_ap.add_parser("build",
+                                 help="build snapshot from Makefile project")
+    build_ap.add_argument("source_dir",
+                          help="project's root directory")
+    build_ap.add_argument("output_dir",
+                          help="output directory of the snapshot")
+    build_ap.add_argument("functions_list", nargs='?',
+                          help="list of functions to compare")
+    build_ap.add_argument("--make", help="make tool used to be used for build",
+                          default="make")
+    build_ap.add_argument("--makefile", help="filename of Makefile to be used\
+                          for build")
+    build_ap.add_argument("--clang", help="clang compiler to be used for\
+                          building", default="clang")
+    build_ap.add_argument("--clang-append", help="option to append to clang",
+                          action='append')
+    build_ap.add_argument("--clang-drop", help="option to drop from clang",
+                          action='append')
+    build_ap.add_argument("--llvm-link", help="llvm-link to be used for ll\
+                          file linking", default="llvm-link")
+    build_ap.add_argument("--llvm-dis", help="llvm-dis to be used for bc file\
+                          disassembly", default="llvm-dis")
+    build_ap.add_argument("--target", help="Makefile target to build",
+                          action='append')
+    build_ap.add_argument("--reconfigure", help="reconfigure autotools-based\
+                          project with CC=<wrapper>", action="store_true")
+    build_ap.add_argument("--without-rpython",
```

Comment:
I was thinking about autotools configure, which uses `--without-...` to disable features while building, but thinking about it again, this is a different case and it could be confusing, `--no-rpython` is better here.